egs2/TEMPLATE/diar1/diar.sh

```diff
@@ -485,8 +485,7 @@ if ! "${skip_eval}"; then
             _opts+="--config ${inference_config} "
         fi
 
-        # for dset in "${valid_set}" ${test_sets}; do
-        for dset in "${valid_set}"; do
+        for dset in "${valid_set}" ${test_sets}; do
```

Comment:
This might be like this?

Reply:
Thanks for pointing out. I guess it's not intended, and should be as you modified.